src/GridFieldAddNewInlineButton.php

```diff
@@ -86,11 +86,11 @@ public function getHTMLFragments($grid)
 
         $fragment = $this->getFragment();
 
-        if (!$editable = $grid->getConfig()->getComponentByType('GridFieldEditableColumns')) {
+        if (!$editable = $grid->getConfig()->getComponentByType(GridFieldEditableColumns::class)) {
             throw new Exception('Inline adding requires the editable columns component');
         }
 
-        Requirements::javascript(THIRDPARTY_DIR . '/javascript-templates/tmpl.js');
+        Requirements::javascript('gridfieldextensions/javascript/tmpl.js');
```

Comment:
As this is 4.0 compat branch you should be using the new resource locator syntax `symbiote/silverstripe-gridfieldextensions:javascript/tmpl.js`